<launch>
  <arg name="arm" default="PSM2"/>
  <arg name="yaml" default="$(find dvrk_vision)/defaults/registration_params_cmu.yaml"/>
  <include file="$(find dvrk_vision)/launch/just_cams.launch">
  	<arg name="camera_parent" default="$(arg arm)_psm_base_link"/>
  </include>

  <node ns="stereo" pkg="dvrk_vision" type="annotate_images.py"
  		  name="registration" output="screen" required="true">
    <param name="registration_yaml" value="$(arg yaml)" />
  </node>
  		

</launch>